changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1 March 2022 12:00 UTC
+
+* [Customer](mews-api.md#Customer) object extended. The field `title` was added.
```

Comment:
From the structure below it seems that the message should be a little different. Based on line 90 (## 26th March 2020 12:00 UTC)
```suggestion
* [Customer](mews-api.md#Customer) extended. The `title` is now supported.
```